docs/source/fetching/persisted-queries.mdx

```diff
@@ -0,0 +1,98 @@
+
+Each query or mutation is identified by the SHA256 hash of its contents that can be sent to the server in lieu of sending a full query document. For APQs specifically, if the hash can't be found by the server, it sends back an error indicating that it needs the full query. If it receives this specific error, Apollo iOS will automatically retry the operation with the full query document without you having to do anything.
+
+### Support
+
+| | **Persisted Queries** | **Automatic Persisted Queries** |
+| - | :---------------: | :-------------------------: |
+| **GraphOS** | Yes | No |
+| **Apollo Server** | Yes?, version ?? | Yes, version ?? |
+
+### Feature
+
+| | **Persisted Queries** | **Automatic Persisted Queries** |
+| - | :---------------: | :-------------------------: |
+| **Pre-registered queries** | Yes | Yes |
+| **Automatically persist queries from client** | No | Yes |
+| **Allow only safelisted persisted queries** | Yes | Yes |
+| **Allow both persisted and non-persisted queries** | Yes | Yes |
+
+## Using Persisted Queries with Apollo iOS
+
+To enable persisted queries you will need to configure your code generation and `ApolloClient` correctly.
+
+### 1. Configure code generation for Persisted Queries
+
+To use persisted queries, your operations need to have a SHA256 hash identifier included in the generated operation objects. To configure code generation you will need to set the following 2 options in your `ApolloCodegenConfiguration` JSON file:
+
+In the `output` section of your configuration you will need to set the [`operationManifest`](./../code-generation/codegen-configuration#operation-manifest) with a version of either `persistedQueries` or `legacyAPQs`, and a path to output the manifest file.
+
+```json title="apollo-codegen-config.json"
+"output": {
+    "operationManifest" : {
+        "path" : "./generated/operationIdentifiers.json",
+        "version" : "persistedQueries"
+    }
+}
+```
+
+Then in the `options` section of your configuration you will need to set the `operationDocumentFormat` to be either `definition`, `operationId`, or both `definition` and `operationId`. This will determine how to generate the operation documents for your generated operations.
+
+```json title="apollo-codegen-config.json"
+"options": {
+    "operationDocumentFormat" : [
+        "definition",
+        "operationId"
+    ]
+}
+```
+
+The resulting operation manifest json would like this: 
+
+```json title="operationIdentifiers.json"
+{
+  "format": "apollo-persisted-query-manifest",
+  "version": 1,
+  "operations": [
+    {
+      "id": "e0321f6b438bb42c022f633d38c19549dea9a2d55c908f64c5c6cb8403442fef",
+      "body": "query GetItem { thing { __typename } }",
+      "name": "GetItem",
+      "type": "query"
+    }
+  ]
+}
+```
+
+> **Note:** This example is using the `persistedQueries` version of the operation manifest, the `legacyAPQs` version will look slightly different.
+
+### 2. Enable persisted queries on your `ApolloClient`
```

Comment:
Should this say "automatic" persisted queries? I figure if folks have done the above manifest input then they don't need to use `autoPersistQueries`, right?

Reply:
So right now in iOS the `autoPersistQueries` flag is used to enable both PQs and APQs for the network code, eventually we will add a separate flag specifically for PQs though.